src/components/boards/Board.css

```diff
@@ -0,0 +1,114 @@
+.itemImage {
+  height: auto;
+  width: 100%;
+  margin-bottom: 2%;
+  border-top-left-radius: 10%;
+  border-top-right-radius: 10%;
+}
+
+.cards {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  margin: 2%;
+}
+
+.oneCard {
+  width: 200px;
+  height: auto;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+  background-color: #ade8f4;
+  border-radius: 10%;
+  margin: 2%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.oneCard a {
+  color: black;
+  text-decoration: none;
+  margin-bottom: 2%;
+}
+
+.oneCard:hover {
+  box-shadow: 0 12px 16px 0 rgba(46, 42, 42, 0.767);
+}
+
+.categories {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  margin: 2%;
+}
+
+#oneCategory {
+  height: 120px;
+  width: 50%;
+  color: black;
+  font-family: Impact, Haettenschweiler, "Arial Narrow Bold", sans-serif;
+  font-size: 20px;
+  border-radius: 20px;
+}
+#oneCategory:hover {
```

Comment:
Firstly good work! As @udenizdemirbilek pointed out, we are using class, not id. Other than that everything looks very nice Ayshe, ellerine sağlık :)